homeassistant/components/metoffice/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
```

Comment:
A good improvement would also be to move all the entity specifics that currently are in `native_value` to the entity_descriptions